app/src/main/java/org/oppia/app/utility/Logger.kt

```diff
@@ -0,0 +1,122 @@
+package org.oppia.app.utility
+
+import android.content.Context
+import android.util.Log
+import org.oppia.app.application.ApplicationContext
+import org.oppia.app.application.OppiaApplication
+
+import java.io.BufferedWriter
+import java.io.File
+import java.io.FileOutputStream
+import java.io.FileWriter
+import java.io.IOException
+import java.io.PrintStream
+import java.text.SimpleDateFormat
+import java.util.Calendar
+import java.util.Locale
+import javax.inject.Inject
+import kotlin.coroutines.coroutineContext
+import android.widget.Toast
+
+/** Wrapper class for Android Log. */
+class Logger @Inject constructor(@ApplicationContext context: Context) {
+
+  private val ENABLE_CONSOLE_LOG = true
+  private val ENABLE_FILE_LOG = true
+  private val GLOBAL_LOG_LEVEL = LogLevel.VERBOSE
+  private val LOG_DIRECTORY =File(context.filesDir, "oppia_app.log")
```

Comment:
Nit: space before ``File`` for consistency.

Reply:
done